schema.sql

```diff
@@ -0,0 +1,10 @@
+/* CREATING THE ANIMALS TABLE */
+
+CREATE TABLE animals (
+   id int,
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you mark the `id` column as a `PRIMARY KEY` to make it unique in the table. **[Here](https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-primary-key/)** is an example of how to define a column as a **Primary Key** in a table.